spec/filters/alter_spec.rb

```diff
@@ -1,6 +1,6 @@
 # encoding: utf-8
 
-require "logstash/devutils/rspec/spec_helper"
+require_relative '../spec_helper'
```

Comment:
I made the update based on what is generated by logstash-plugin from logstash 5.0.0, didn't notice this change.
Should it be reverted?